rootfs/scheduler/__init__.py

```diff
@@ -643,15 +643,31 @@ def _set_container(self, namespace, container_name, data, **kwargs):  # noqa
             self._default_readiness_probe(data, kwargs.get('build_type'), env.get('PORT', None))
 
     def _get_private_registry_config(self, registry, image):
-        # try to get the hostname information
-        hostname = registry.get('hostname', None)
-        if not hostname:
-            hostname, _ = docker_auth.split_repo_name(image)
-        if hostname == docker_auth.INDEX_NAME:
-            hostname = "https://index.docker.io/v1/"
+        secret_name = settings.REGISTRY_SECRET_PREFIX
```

Comment:
can we remove this? as far as I can tell `secret_name` cannot be changed, so we might as well reference `settings.REGISTRY_SECRET_PREFIX` directly rather than waste memory allocating a local variable.